.github/workflows/release-checksums.yml

```
@@ -47,13 +53,15 @@ jobs:
     runs-on: ubuntu-24.04
+    needs:
+      - release_info
     outputs:
       file: sha512sums.txt
     steps:
       - uses: robinraju/release-downloader@v1.12
         with:
-          repository: ${{ env.GITHUB_REPOSITORY }}
-          latest: true
+          repository: ${{ github.repository }}
+          tag: ${{ steps.release-info.outputs.release-tag }}
```

actionlint (1.7.4)

property "release-info" is not defined in object type {}

(expression)